cpp/ql/src/semmle/code/cpp/exprs/ComparisonOperation.qll

```diff
@@ -134,3 +134,17 @@ class LEExpr extends RelationalOperation, @leexpr {
 
   override Expr getLesserOperand() { result = getLeftOperand() }
 }
+
+/**
+ * A C/C++ 'spaceship' operation
+ * ```
+ * auto c = (a <=> b);
+ * ```
+ */
+class SpaceshipExpr extends BinaryOperation, @spaceshipexpr {
```

Comment:
I'm a bit torn about what this class should be named and what file it should be declared in. I've asked for input at https://github.slack.com/archives/CP6990YTU/p1583828309146800.

Reply:
Discussion in Slack seems to suggest that this should be called `SpaceshipExpr`.

@jbj: Do you have any view on which file this should be declared in?

Reply:
I agree that we decided on `SpaceshipExpr`.

I don't have a strong opinion on the file -- nobody's supposed to import those files directly. I think I'd put it in `Expr.qll` since the other possible files seem to be associated with positions in the class hierarchy that aren't right for `SpaceshipExpr`.

Reply:
`SpaceshipExpr` now lives in `Expr.qll`.